libs/perun/services/src/lib/mfa-handler.service.ts

```diff
@@ -155,8 +155,8 @@ export class MfaHandlerService {
 
   private setupMfaWindow(): Window {
     const url = location.pathname + location.search;
-    const width = 600;
-    const height = 600;
+    const width = window.outerWidth * 0.6;
```

Comment:
This sets the size of the new window based on the size of the existing window, but wouldn't setting it based on the monitor size make more sense? An already small windowed window opening an even smaller window doesn't seem practical.
I think this can be achieved with `window.screen.width` and `window.screen.height`.